package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@ibm-skills-network/olx-cli",
   "description": "A command line tool for Open Learning XML",
-  "version": "0.1.14",
+  "version": "0.1.16",
```

Comment:
Looks like 0.1.16 is already published on npm (https://www.npmjs.com/package/@ibm-skills-network/olx-cli) so this will need to be bumped again (and in the readme)

(unless 0.1.16 is the changes in this PR)